### DISCUSSION TITLE
Can I hide the empty bottom bar?

### DISCUSSION THREAD

**Discussion by kkarakondjull** [florisboard/florisboard#1233] [Q&A] [answered]
![Screenshot_20210906-222638](https://user-images.githubusercontent.com/90217157/132255940-b4c55e91-becf-44e3-98a6-ecbaa3072512.jpg)
How do I get rid of it please! I'll switch from ASK to Floris if I can!
Ps. First time using GitHub, hope this is the right place.

**Comment by patrickgold** [comment]
The marked bottom bar is the Android navigation bar which is present on most screens - to me it seems like the icon color matches the background, thus it seems empty. I don't know if I can even consistently hide this bar, as it is mostly managed by the system. Do the buttons still exist (back, home and app overview) or does nothing happen if you touch the empty space of the bar?

**Reply by kkarakondjull** [reply]
Thank you for replying! I don't use virtual buttons, only swipe gestures for navigation. Which is why it's confusing that this would show up!

**Reply by kkarakondjull** [reply]
Now that I checked, the default Gboard on my phone also has it. I guess it's something to do with my Android version (OnePlus 7 Pro running latest 11 stable), because it didn't use to be there (on this same phone) and my friend's phone doesn't have it either (Xiaomi Mi 9).

**Comment by kkarakondjull** [comment, marked as answer]
I removed it with this Magisk Module
https://github.com/Magisk-Modules-Repo/HideNavBar
